src/courseware/course/sequence/Sequence.test.jsx

```diff
@@ -329,32 +330,20 @@ describe('Sequence', () => {
       await waitFor(() => expect(screen.queryByText('Loading learning sequence...')).not.toBeInTheDocument());
 
       screen.getAllByRole('button', { name: /previous/i }).forEach(button => fireEvent.click(button));
-      expect(testData.previousSequenceHandler).toHaveBeenCalledTimes(2);
+      expect(testData.previousSequenceHandler).toHaveBeenCalledTimes(1);
       expect(testData.unitNavigationHandler).not.toHaveBeenCalled();
 
       screen.getAllByRole('button', { name: /next/i }).forEach(button => fireEvent.click(button));
-      expect(testData.nextSequenceHandler).toHaveBeenCalledTimes(2);
+      expect(testData.nextSequenceHandler).toHaveBeenCalledTimes(1);
       expect(testData.unitNavigationHandler).not.toHaveBeenCalled();
 
       expect(sendTrackEvent).toHaveBeenNthCalledWith(1, 'edx.ui.lms.sequence.previous_selected', {
-        current_tab: 1,
-        id: testData.unitId,
-        tab_count: 0,
-        widget_placement: 'top',
-      });
```

Comment:
[nit]: Previously this test checked the switching functionality for both the top and bottom navigation blocks.
I would not like to remove the existing logic for checking any code. Can we consider adding/changing the test coverage exclusively for the changes you added?